tests/spec/jquery.liteuploader.spec.js

```diff
@@ -81,13 +79,29 @@ describe('Lite Uploader', function () {
             expect(liteUploader._performUpload).not.toHaveBeenCalled();
         });
 
+        it('should not proceed with upload if beforeRequest was rejected', function () {
```

Comment:
New test to make sure `beforeRequest` stops the upload, if the client operation fails.
The success or resolved state is covered in the 'should proceed with upload if no errors are found' test.